pkg/env/prod.go

```diff
@@ -261,6 +273,19 @@ func (p *prod) ACRDomain() string {
 	return p.acrDomain
 }
 
+func (p *prod) OIDCStorageAccountName() string {
```

Comment:
My main concern is we'll have to maintain the storage account name in two separate places. Ex: https://msazure.visualstudio.com/AzureRedHatOpenShift/_git/ARO-Pipelines?path=/rp/oidc/Region/Templates/modules/storage.bicep

Would you be good if we:
- Define the storage account names in `rp-config`
- Have ARO-Pipelines and ARO-RP consume that name from `rp-config`

Reply:
Thanks Nic for raising this, I agree and back your idea for the consistency. It was hard anyways to get the container naming using the assumption present in the function.
Although I would like to get the consensus on the this.
cc: @cadenmarchese

Reply:
As per discussion, this is going to be handle as a separate issue after the PR is merged. I will post the Jira story for the same here when created.

Reply:
Created a story for the same:- https://issues.redhat.com/browse/ARO-8405